src/main/java/org/dependencytrack/resources/v1/TagResource.java

```diff
@@ -124,6 +131,98 @@ public Response getTaggedProjects(
         return Response.ok(tags).header(TOTAL_COUNT_HEADER, totalCount).build();
     }
 
+    @POST
+    @Path("/{name}/project")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Tags one or more projects.",
+            description = "<p>Requires permission <strong>PORTFOLIO_MANAGEMENT</strong></p>"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "204",
+                    description = "Projects tagged successfully."
+            ),
+            @ApiResponse(
+                    responseCode = "404",
+                    description = "A tag with the provided name does not exist.",
+                    content = @Content(schema = @Schema(implementation = ProblemDetails.class), mediaType = ProblemDetails.MEDIA_TYPE_JSON)
+            )
+    })
+    @PermissionRequired(Permissions.Constants.PORTFOLIO_MANAGEMENT)
```

Comment:
As of #800, we need to consider fine-grained permissions. In this case, we should allow `PORTFOLIO_MANAGEMENT_UPDATE` in addition to `PORTFOLIO_MANAGEMENT`.